pytorch_lightning/loggers/trains.py

```diff
@@ -23,7 +23,9 @@ def any_lightning_module_function_or_hook(...):
         self.logger.experiment.whatever_trains_supports(...)
 
 """
-
+import PIL
+import numpy
+import pandas
```

Comment:
no, we do not want to add Pandas to requirements just because of one type of reference...
We shall find another way of it rise error...

Reply:
I agree. I pushed a new commit, let me know what you think.

Reply:
I take it back, they have pandas only as optional dep. Maybe best is to just remove it from type annotation. Otherwise, if we leave it, docs give error.

Reply:
ok, let's drop it thow